docs/nginx.md

```diff
@@ -3,4 +3,44 @@ id: nginx
 title: nginx config
 ---
 
-todo
+You may want to use your nginx server as a reverse proxy to run gotify.
+
+Here is a sample config file if you run your gotify instance on port 1245
+
+```yaml
```

Comment:
I guess this isn't yaml. Could you remove it?

Reply:
Yes, my bad I set it to nginx